fulp_modules/main_features/bloodsuckers/code/bloodsucker/bloodsucker_life.dm

```diff
@@ -394,11 +392,11 @@
 	if(!clan.bloodsucker_sunlight.amDay && total_damage >= 10 && !HAS_TRAIT(owner.current, TRAIT_NODEATH))
 		Torpor_Begin()
 
-/datum/antagonist/bloodsucker/proc/Check_End_Torpor()
-	/// You're not in Sol? (Slept in a Locker for example), then you don't need to leave it.
+/datum/antagonist/bloodsucker/proc/Check_End_Torpor(SkipChecks = FALSE)
```

Comment:
why are you adding skipchecks if it doesnt do anything? we dont want it to do anything, either

Reply:
I added it before reformatting the previous code and just forgot about it, I'll make sure to remove it